docs/sdk/block.md

```diff
@@ -162,4 +162,73 @@ subscription.subscribe()
 
 // unsubscribe
 subscription.unsubscribe()
+```
+
+### WebSocket Support
+
+Alephium full node also supports streaming blocks using
+WebSocket. Here is an example using
+[wscat](https://github.com/websockets/wscat) from terminal:
```

Comment:
Let's add a remark that this is under active development, and breaking changes might be expected.